magda-registry-aspects/organization-details.schema.json

```diff
@@ -19,6 +19,10 @@
             "title":
                 "The URL of an image representing the organization, such as its logo.",
             "type": "string"
+        },
+        "datasetCount": {
```

Comment:
Surely we don't want to store this in the registry? It's too dynamic?

Reply:
This file shouldn't be changed. 
Adam did try a potential solution that relies on DB queries. 
But we eventually go with elasticsearch
Will remove that.